.github/workflows/canister-tests.yml

```diff
@@ -263,7 +290,7 @@ jobs:
           mkdir dist
           touch dist/index.{html,css,js}
           touch dist/index2.js
-          cargo test
+          cargo test --features 'custom_origin'
```

Comment:
If you don't want to flip logic of the feature flag, please copy vc_demo_issuer_custom_origin.wasm.gz to vc_demo_issuer.wasm.gz before running the tests, so that the tests can find the wasm.

Reply:
> If you don't want to flip logic of the feature flag

I really don't care! Up to you. This was the easiest thing to do with what I had.